readme.md

```diff
@@ -264,6 +264,9 @@ The following bash scripts are useful when working on this project:
     ```sh
     RUST_BACKTRACE=1 cargo test
     ```
+    ```sh
+    RUST_BACKTRACE=1 cargo test --features json
+    ```
```

Comment:
Do you recommend that humans run both commands? One or the other?

Reply:
Can be replaced with 'cargo test --all-features'

Reply:
ok, can you do that? One test example, with `all-features`?

Reply:
done